packages/bigquery/src/index.js

```diff
@@ -66,7 +66,7 @@ function BigQuery(options) {
   }
 
   var config = {
-    baseUrl: 'https://www.googleapis.com/bigquery/v2',
+    baseUrl: 'https://www.googleapis.com./bigquery/v2',
```

Comment:
I also disagree that the benefit is tiny, in our case. We see a large number of dropped requests due to this issue. See my response in the issue #2213 for more details on how it's affecting our organization.